bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/model/jsonadapter/EvaluationLatencyMetricsEventAdapterFactory.kt

```diff
@@ -0,0 +1,101 @@
+package io.bucketeer.sdk.android.internal.model.jsonadapter
+
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.JsonReader
+import com.squareup.moshi.JsonWriter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+import io.bucketeer.sdk.android.BKTException
+import io.bucketeer.sdk.android.internal.model.MetricsEventData
+import java.lang.reflect.Type
+
+class EvaluationLatencyMetricsEventAdapterFactory : JsonAdapter.Factory {
```

Comment:
Can you add tests for this class?

Reply:
Do you know if `GetEvaluationLatencyMetrics` is still being used? @yshrsmz 
I'm just not sure if it is a waste for adding more tests for something that will be not used anymore 

Please check my comment here https://github.com/bucketeer-io/android-client-sdk/issues/56#issuecomment-1565299243

@yshrsmz

Reply:
https://github.com/bucketeer-io/android-client-sdk/issues/56#issuecomment-1565832276

commented. Not 100% sure but should be fine.

@cre8ivejp let us know if you want to keep the implementation.

Reply:
@yshrsmz 
CC: @duyhungtnn

Thank you for commenting.

The `GetEvaluationLatencyMetrics` is still in use, but because we are going to remove it and I'm not releasing this PR until this [issue](https://github.com/bucketeer-io/android-client-sdk/issues/56) is merged, it's fine not writing the tests for those cases.

Reply:
thank you @cre8ivejp  for confirming.